lib/manager/gitlabci/index.ts

```diff
@@ -1,9 +1,9 @@
 import { LANGUAGE_DOCKER } from '../../constants/languages';
-import { extractPackageFile } from './extract';
+import { extractAllPackageFiles, extractPackageFile } from './extract';
 
 const language = LANGUAGE_DOCKER;
 
-export { extractPackageFile, language };
+export { extractAllPackageFiles, extractPackageFile, language };
```

Comment:
```suggestion
export { extractAllPackageFiles, language };
```

Only one of those functions should be exported

Reply:
I tried that, but if I do this, renovate will not work anymore.
This is because the `doAutoReplace` in `lib/workers/branch/auto-replace.ts` does call `confirmIfDepUpdated` and this function does use the `extractPackageFile` function:
https://github.com/renovatebot/renovate/blob/cda66e08c533c400e8a44ccd23af31cd7e7de6ef/lib/workers/branch/auto-replace.ts#L24-L32

Either `confirmIfDedpUpdated` has to be refactored or `extractPackageFile` needs to be exported – like the `batect` manager:
https://github.com/renovatebot/renovate/blob/cda66e08c533c400e8a44ccd23af31cd7e7de6ef/lib/manager/batect/index.ts#L1-L3

Reply:
@viceice @rarkins What is the preferred way to do it?

Reply:
leave this as is for now